docs/api.md

```diff
@@ -25,18 +25,18 @@ For information about the Antrea API versioning policy, please refer to this
 These are the CRDs currently available in `crd.antrea.io`.
 
 | CRD | CRD version | Introduced in | Deprecated in / Planned Deprecation | Planned Removal |
-|---|---|---|---|---|
-| `AntreaAgentInfo` | v1beta1 | v1.0.0 | N/A | N/A |
-| `AntreaControllerInfo` | v1beta1 | v1.0.0 | N/A | N/A |
-| `ClusterGroup` | v1alpha3 | v1.1.0 | N/A | N/A |
-| `ClusterNetworkPolicy` | v1alpha1 | v1.0.0 | N/A | N/A |
-| `Egress` | v1alpha2 | v1.0.0 | N/A | N/A |
-| `ExternalEntity` | v1alpha2 | v1.0.0 | N/A | N/A |
-| `ExternalIPPool` | v1alpha2 | v1.2.0 | N/A | N/A |
-| `Group` | v1alpha3 | v1.8.0 | N/A | N/A |
-| `NetworkPolicy` | v1alpha1 | v1.0.0 | N/A | N/A |
-| `Tier` | v1alpha1 | v1.0.0 | N/A | N/A |
-| `Traceflow` | v1alpha1 | v1.0.0 | N/A | N/A |
+|---|---|---------------|---|---|
```

Comment:
Why is this changed? I think you should just modify the line for Egress only, otherwise, it's easy to introduce conflicts with other API promotion PRs.